_release/bat/workload_bat/workload_bat_test.go

```diff
@@ -115,6 +112,23 @@ func testCreateWorkload(t *testing.T, public bool) {
 	if w.Name != opt.Description || w.CPUs != opt.Defaults.VCPUs || w.Mem != opt.Defaults.MemMB {
 		t.Fatalf("Workload not defined correctly")
 	}
+
+	// delete the workload. We only support delete of public workloads.
```

Comment:
This comment is confusing. You delete a non-public workload in the second part of the if.